platform-mbean/src/main/java/org/jboss/as/platform/mbean/RuntimeMXBeanAttributeHandler.java

```diff
@@ -116,7 +116,11 @@ static void storeResult(final String name, final ModelNode store) {
         } else if (PlatformMBeanConstants.BOOT_CLASS_PATH_SUPPORTED.equals(name)) {
             store.set(ManagementFactory.getRuntimeMXBean().isBootClassPathSupported());
         } else if (PlatformMBeanConstants.BOOT_CLASS_PATH.equals(name)) {
-            store.set(ManagementFactory.getRuntimeMXBean().getBootClassPath());
+            if (!ManagementFactory.getRuntimeMXBean().isBootClassPathSupported()) {
+                store.set(""); // JDK9 and above does not have boot class path
+            } else {
+                store.set(ManagementFactory.getRuntimeMXBean().getBootClassPath());
+            }
```

Comment:
How are the semantics here different from JDK 8, where this call could throw UnsupportedOperationException?

The way these resources handle these cases are if read-attribute is invoked reading an attribute like this, the UnsupportedOperationException is caught and an OperationFailedException is thrown. If read-resource is invoked, we catch the UnsupportedOperationException and return undefined as the attribute's value in the overall response. We fail on read-attribute because it's a direct request where we behave consistently with what the RuntimeMXBean does with a direct request. There is no RuntimeMXBean equivalent to read-resource, so we do what's logical when reading numerous attributes and aggregating.  

I don't want to deviate from that pattern for particular attributes unless it's absolutely necessary.